### DISCUSSION TITLE
Questions about running G-Helper with Armoury Crate still installed

### DISCUSSION THREAD

**Discussion by TrentEsq** [seerge/g-helper#917] [General]
Hello!
I have a Strix G18 (i9-13980HX & RTX 4080) and have a few questions before I give the app a go,

I plan to keep Armoury Crate installed (MyAsus is uninstalled) to avoid any issues (potentially) occuring after a Bios update/flash. I would think it's very highly unlikely that any conflict/issue occurs after booting to OS due to Armoury Crate being absent after a Bios update, however Asus is Asus and it took a couple Bios updates for faster than 4800 Ram speeds to be usable, as well as for them to unlock the Cpu for extended undervolting. Plus, since they don't include a changelog whatsoever with their Bios updates I'd rather play it safe and just disable Armoury in the case it gets re-installed/set to re-install via a future Bios update.

So the plan is - start/run g-helper, configure g-helper to start at boot, disable Armoury Crate Service via Services App in Windows, disable all Asus tasks in Task Scheduler, reboot.

Would g-helper pull the currently applied settings when being ran for the first time? I have Armoury set to the 'Performance' profile, so would g-helper mirror/reflect as such?

I noticed the three profile settings in g-helper are *Silent*, *Balanced*, & *Turbo*, but Armoury is *Silent*, *Performance*, & *Turbo*
Does 'Balanced' = 'Performance'? As in, does it pull from the 'Performance' profile settings in Bios to utilize for the 'Balanced' profile?

Since I have 'Performance' set in Armoury and in Windows Power Plan as well, would an issue arise from runnnig g-helper for the first time with those settings already in effect beforehand? I ask because in the 'Recommendations' section it's strongly recommended to set Windows to the 'Balanced' power plan when running g-helper, is that for a critical reason?

I know all of that sounds roundabout as opposed to just following the g-helper setup guide as it has been written, but it's been a bit of a task setting up this notebook to a point where I'm happy with it (undervolting, looking up what all the Asus bloat/services/scheduled tasks do and disabling them, figuring out bootloops after updating Bios, etc.), and any answers given would be much appreciated!

**Comment by seerge** [comment]
@TrentEsq hello,

You can try and just run app along with  AC, it's ok. I just recommend to set windows power plan to Balanced (as it's not limiting PC anyhow, and most optimal). Just in case AC will override power plan back to it's own as soon as you set something in it.

If you will set modes / any settings in G-Helper it will override what was set by AC prior to that until AC overrides them back.

**Reply by TrentEsq** [reply]
Got it, thank you for the reply!
I completed the steps as I described - ran G-Helper, disabled all Armoury items in Services & Task Scheduler, changed Windows power plan to 'Balanced', rebooted. All went smoothly/without issue & all functions seem to work properly.
(Except that even having disabled 'Asus Smart Display Control' from auto-running in Task Scheduler, it still persisted in auto-running even after restart/reboot, so I had to manually terminate it in Task Manager)

Two more questions,
I leave the charger plugged in at all times, never use sleep/hibernate, perform a full shutdown when not using the notebook, have disabled 'Modern Standy'(S0 Low Power Idle) via RegEdit, & have disabled FastBoot in Bios. Will the 'Battery Charge Limit' function persist even when the notebook is shutdown, as it did with the MyAsus 'Battery Care' function?

In G-Helper - 'Extra Settings-->Other-->Log'
What does the 'AutoSetting for Online' correspond to?

Again, thank you for your time and the awesome app! =)

**Reply by seerge** [reply]
Battery charge limit is same setting in bios as myasus sets

Online in log = plugged in

**Reply by TrentEsq** [reply]
Got it, thank you!

I was looking at the fan curves of each profile in G-Helper, and I noticed that as I switched between them, it was also switching the gpu mode in real-time. I didn't think anything of it and switched back to 'Ultimate', however when I booted up a game I noticed that some effects were no longer rendering properly and most noticeably, all blacks/dark colours were rendered as if at a lower resolution/blocky/pixelated. I checked resolution settings in Nvidia Control Panel, as well as in Windows Display settings, and everything looked normal.
I then switched to the 'standard' gpu mode, rebooted, and upon booting the same game, everything was rendering correctly again, switched back to 'ultimate', rebooted, the problem returned. I updated the Nvidia driver, but the problem persists.
Do you have any insight as to what could be going on with the 'ultimate' gpu mode on my notebook for it not to be rendering colours/effects properly? Maybe a setting that G-Helper changes that I should look for?

Thanks in advance!

**Reply by seerge** [reply]
Switching between performance modes (top row) doesn't touch anything about GPU.

If you mean GPU modes - they obviously turn dGPU on or off. I wold advice to check readme about all this https://github.com/seerge/g-helper 

Ultimate mode just routes your internal display to dGPU, but since you have advanced optimus you don't really need it in a first place. You can safely keep GPU in Standard all the time.

G-Helper doesn't touch (and can't really) touch any driver settings for your GPU tho :) (with an exception of slight overclocking under fans+power, but only if you change it there). All this settings are same as in Armoury Crate.

**Reply by TrentEsq** [reply]
Understood, & the cause looks to have been an update to 'Asus Control Center Interface' I applied recently. The timing of it all worked out to the point where I didn't notice the colour-banding until after disabling Armoury/MyAsus & getting started with G-Helper.

Apologies for considering G-Helper as a potential cause of the issue, & Thank You for all your help and time thusfar.
Asus would do well to notice your skills & contact you to offer a paying position/contracting work to develop software for them.

**Comment by BBarfryyy** [comment]
Hello,
i discovered Ghelper recently, i love how its simple and fast. However, I noticed that I cant handle RGB with it, except on startup... 
I have a rog strix G15 and AC still installed. Did you noticed somthing like this too ?
Thanks

**Reply by seerge** [reply]
It could be that AC services override backlight settings (constantly)

So it’s better to stop them (Aura and Lightning services in windows Services app) and check if you can set RGB via G-Helper after :)

**Reply by BBarfryyy** [reply]
i already tried, but i still have aura installed, ill try to uninstall it.
EDIT : okay what you said works, thank you a lot ! Do you know if you will handle aura with external peripherics ? like keyboards or mouse.
Thanks, you software is a bomb <333

**Reply by AraeAryrha** [reply]
> i already tried, but i still have aura installed, ill try to uninstall it. EDIT : okay what you said works, thank you a lot ! Do you know if you will handle aura with external peripherics ? like keyboards or mouse. Thanks, you software is a bomb <333

Hi,

Could you share your full setup for G Helper? I am also using the ROG Strix G15, but my games keep closing automatically, and I don't know how to fix it. Please help me!

Thank you!

**Reply by BBarfryyy** [reply]
I can't for now as I dont have my laptop with me, but i'll share it to you asap, although I don't think your problem come from g helper.
What do you mean by they close automatically ?

**Reply by AraeAryrha** [reply]
> I can't for now as I dont have my laptop with me, but i'll share it to you asap, although I don't think your problem come from g helper. What do you mean by they close automatically ?

If I uninstall G Helper, the games work fine, and I can play longer, but the temperature is quite high. However, when I use G Helper again, the games automatically close. In Valorant, I can only play half a round before it says my graphics driver is too old, even though I have updated to the latest version. For The First Descendant, I can play without G Helper, but when I try using it, the game can't even reach the menu section and closes automatically without any error message. I followed a Redditor's setup for my G Helper since my knowledge about this is not very good.

**Reply by seerge** [reply]
@AraeAryrha what settings are you using (post a full screenshot here including ``Fans + Power`` opened)? Try to use all defaults, just ``Standard / Balanced``.

If game says drivers are old, then it's probably the case or something is wrong with the drivers (as G-Helper is not a driver and it's not interacting with that anyhow in a first place)

**Reply by AraeAryrha** [reply]
> @AraeAryrha what settings are you using (post a full screenshot here including `Fans + Power` opened)? Try to use all defaults, just `Standard / Balanced`.
> 
> If game says drivers are old, then it's probably the case or something is wrong with the drivers (as G-Helper is not a driver and it's not interacting with that anyhow in a first place)

This is my balance setting. I still don't know how to set it up perfectly; I just followed some Reddit comments to configure G Helper. I'm really scared to try the advanced settings because I don't know if my device is compatible. For the driver, I have already reinstalled it, but it still detects mine as an old driver. I'm using version 556.12 right now.

My device:

ROG Strix G15 G513IE
Processor: AMD Ryzen 7 4800H with Radeon Graphics, 2901 MHz, 8 Cores, 16 Logical Processors
NVIDIA GeForce RTX 3050 Ti Laptop GPU
16GB RAM
![Balanced cpu](https://github.com/seerge/g-helper/assets/136577727/6f51442b-3e7c-4db9-8ea2-f8c5261858bd)
![Balanced gpu](https://github.com/seerge/g-helper/assets/136577727/6fe87f92-7685-440f-8627-2a873127e859)
![balanced adv](https://github.com/seerge/g-helper/assets/136577727/b307bd2c-1304-4ab0-808a-2e17dfe5f238)

**Reply by seerge** [reply]
@AraeAryrha i see that you have **overclocked your GPU** (and probably too much). 

Try to set both ``Core Clock offset`` and ``Memory Clock offset`` back to 0 (or click ``Factory Defaults`` it should put them to 0 automatically).

And try your game again.

If it helps - you can try to overclock again, but with smaller amounts :)

**Reply by AraeAryrha** [reply]
> @AraeAryrha i see that you have **overclocked your GPU** (and probably too much).
> 
> Try to set both `Core Clock offset` and `Memory Clock offset` back to 0 (or click `Factory Defaults` it should put them to 0 automatically).
> 
> And try your game again.
> 
> If it helps - you can try to overclock again, but with smaller amounts :)

Okay, I have already factory reset both of them. However, when I try playing Solo Leveling, it reaches 86°C and 66W on average with medium graphics settings. Is there any method to reduce the temperature further? These temperatures used to occur only when I was still using AC. What are some small adjustments I can try?

**Reply by seerge** [reply]
@AraeAryrha **so it helped ?** And games are running fine now ?

86C for CPU or GPU ? For CPU it's fine as it can go up to 96C before it starts to drop power.

**Reply by AraeAryrha** [reply]
@seerge Okay, I have already tried it, and there were no crashes for the game. For The First Descendant with medium graphics, my average FPS is 54-67, and the GPU temperature is 86-92°C. For Valorant with medium graphics, I can play with around 96-112 FPS, and the GPU temperature is 88-93°C. Are there any settings to make the temperature a little bit cooler? Because it exceeds the temperature I had when using AC before. People suggest using G Helper to make the device temperature cooler. I hope you can suggest the full settings that are suitable for my device.

**Reply by seerge** [reply]
@AraeAryrha then you have your answer :) You have overclocked GPU too much before. I think it would be good if you would update all your topics on reddit.

As for your temps. All modes are baked into BIOS, so if you use defaults - everything will run probably same as it would with AC :) 

You can try :

- Use `Turbo` instead of `Balanced`, then fans will run faster
- Or set more aggressive fan curves (hold `Shift` and drag whole thing up a bit) + click `Apply custom fan curves`
- Set limits to CPU power for example to 30W-35W (or even less, try what works) + click `Apply power limits`

You can also limit your GPU a bit by setting `Core Clock Limit` to some value. But that will cost you some performance :) 

P.S. You can still try to overclock GPU, but with smaller values (try something like +50Mhz for `Core Clock`)

**Reply by BBarfryyy** [reply]
As you I got 98 °C on CPU ans GPU while in game, I Guess it is normal for laptops. 
I use a 25 W CPU config, agressive fan curves and GPU overclocking and I get little bit better temps.

**Reply by AraeAryrha** [reply]
@seerge Is it like this? Can you check if I'm doing it right? For GPU mode, is it better to use standard? Can you check if the CPU wattage is okay for gaming? For the CPU Boost option, which option should I choose? Many Redditors said disabling it is better. And did I set up the GPU correctly? I tried searching on Google for the ideal CPU temp limit and undervolting. I hope I did it right. If not, please point out which part I made a mistake on. Do I need to tick the "Apply power limit" box?
![Adv cpu](https://github.com/seerge/g-helper/assets/136577727/968ef7f1-25f8-485f-af0a-8a26ace60960)
![Adv gpu](https://github.com/seerge/g-helper/assets/136577727/dfbf3384-b9d6-41ad-8358-2be6783dcbd6)
![Adv adv](https://github.com/seerge/g-helper/assets/136577727/fd01738e-a659-48b8-8577-d3521268fb81)

**Reply by AraeAryrha** [reply]
> As you I got 98 °C on CPU ans GPU while in game, I Guess it is normal for laptops. I use a 25 W CPU config, agressive fan curves and GPU overclocking and I get little bit better temps.

Oh, is it normal? I thought reaching 98°C was bad. How many FPS do you usually get when playing a game?

**Reply by seerge** [reply]
@AraeAryrha it looks ok :) 

- You can limit CPU even more later (literally limit it until you see actual drop in performance)
- I see that you have undervolted CPU in `Advanced`. It's fine, but if game will crash you can set it back to 0 (as it pretty much says in a red warning box)

**Reply by AraeAryrha** [reply]
@seerge Oh, okay, thanks for the advice. The other settings are good, right? The problem might just happen if I change settings in the advanced section.

**Reply by BBarfryyy** [reply]
> > As you I got 98 °C on CPU ans GPU while in game, I Guess it is normal for laptops. I use a 25 W CPU config, agressive fan curves and GPU overclocking and I get little bit better temps.
> 
> Oh, is it normal? I thought reaching 98°C was bad. How many FPS do you usually get when playing a game?


Yeah it is normal on laptops, not on desktops. Lets say that in laptops, components are ""programmed"" to go at this high temps.
I dont know like that but i got 144 FPS on valorant un high (1080p), same for LoL, and 60 on ark survival evolved with mid graphs settings

**Reply by seerge** [reply]
@AraeAryrha its fine.
